kong/dao/schemas_validation.lua

```diff
@@ -185,7 +195,7 @@ function _M.validate_entity(tbl, schema, options)
               end
             end
 
-            if t[column] and type(t[column]) == "table" then
+            if t[column] and t[column] ~= ngx.null and type(t[column]) == "table" then
```

Comment:
It doesn't hurt for such a bloated piece of code to stay explicit after all... Leaving this as-is :)